guide/downloading-dart.rst

```diff
@@ -0,0 +1,60 @@
+Downloading DART
+================
+
+The DART source code is distributed on the GitHub repository
+`NCAR/DART <https://github.com/NCAR/DART>`_ with the documentation
+served through :doc:`readthedocs<../README>`.
+
+Go to https://github.com/NCAR/DART and clone the repository or get the
+ZIP file according to your preference. See the `github help page on
+cloning <https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository>`_
+for more information on how to clone a repository. Take note of the
+directory you installed into, which is referred to as *DARTHOME* throughout 
+this documentation.
+
+.. note::
+
+   If you are interested in contributing to DART, see the
+   :doc:`contributors-guide` for more information. In short, you
+   will need to be familiar with the
+   `GitHub workflow <https://guides.github.com/introduction/flow/>`_.
+
+Unzip or clone the distribution in your desired directory, which we refer to as
+``DART`` in this document. Compiling the code in this tree (as is usually the
```

Comment:
the previous paragraph just finished saying we were going to use DARTHOME ... which is what is used on following sections.

Reply:
nope ... Ben changed everything to use DART ... consistently, since when you clone the repo, you probably get a directory named 'DART'